### PR TITLE
Translate "CVE-2024-35176: DoS in REXML" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translate for #3239
Actual diff is 430f182f5408a978e8cdd529ddbff22d65064ef2

### CODE REVIEW

File: ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: news_post
+title: "CVE-2024-35176: DoS in REXML"
```
Comment (yous) [inline ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md:3]:
```suggestion
title: "CVE-2024-35176: REXML의 DoS 취약점"
```

File: ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+author: "kou"
+translator: "shia"
+date: 2024-05-16 05:00:00 +0000
+tags: security
+lang: ko
+---
+
+REXML gem에서 DoS 취약점이 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-35176](https://www.cve.org/CVERecord?id=CVE-2024-35176)으로 등록되었습니다. REXML gem 갱신을 강하게 추천합니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md:11]:
```suggestion
REXML gem에서 DoS 취약점이 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-35176](https://www.cve.org/CVERecord?id=CVE-2024-35176)으로 등록되었습니다. REXML gem 업그레이드를 강하게 추천합니다.
```

File: ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## 세부 내용
+
+대량의 `<`를 속성값으로 가지고 있는 XML을 파싱할 때, REXML gem은 처리에 시간이 걸립니다.
```
Comment (yous) [inline ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md:15]:
```suggestion
대량의 `<`를 속성값으로 가지고 있는 XML을 파싱할 때, REXML gem은 처리에 긴 시간이 걸립니다.
```

File: ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-05-16-dos-rexml-cve-2024-35176.md]: 🙈

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!

**Pull request merged**